test/customers.test.js

```diff
@@ -29,10 +29,9 @@ describe('ContactHub', () => {
       id: 'foo'
     };
 
-    it('finds an existing Customer', async () => {
+    it('finds an existing Customer', async() => {
       nock(apiUrl)
         .get(`/workspaces/${auth.workspaceId}/customers/${customer.id}`)
-        .query({ nodeId: auth.nodeId })
```

Comment:
is this intended?

Reply:
Yes, it was apparently ignored by previous versions of `nock`. It is not required in this call, and the SDK is correctly not sending it.